src/Mod/TechDraw/App/DrawViewSection.cpp

```diff
@@ -176,11 +176,17 @@ short DrawViewSection::mustExecute() const
 void DrawViewSection::onChanged(const App::Property* prop)
 {
     if (isRestoring()) {
-        DrawView::onChanged(prop);
+        DrawViewPart::onChanged(prop);
         return;
     }
 
     App::Document* doc = getDocument();
+    if (!doc) {
+        //tarfu
+        DrawViewPart::onChanged(prop);
+        return;
+    }
+
     if (prop == &SectionSymbol) {
```

Comment:
You know that this is changing code behaviour, right? ` if (prop == &SectionSymbol) { ... }` will not be reached if `!doc`. while it would before

Reply:
Not to worry.  If there is no doc then things are messed up anyway.